pkg/kubectl/cmd/drain.go

```diff
@@ -113,6 +114,7 @@ func NewCmdCordon(f cmdutil.Factory, out io.Writer) *cobra.Command {
 		},
 	}
 	cmd.Flags().StringVarP(&options.Selector, "selector", "l", options.Selector, "Selector (label query) to filter on")
+	cmd.Flags().StringVarP(&options.Message, "message", "m", options.Message, "Message describing the reason to cordon a node")
```

Comment:
according to [cli convention](https://github.com/kubernetes/community/blob/master/contributors/devel/kubectl-conventions.md#flag-conventions):

> Flag names and single-character aliases should have the same meaning across all commands

Reply:
@shiywang 

I am sorry for not understanding your suggestions. Could you please detail it.
FYI: `-m` or `--message` is added according to `git commit -m` and `docker commit -m`.

Reply:
I mean maybe we should first not having shortcut of this flag because once we add it, it should have same meaning across all kubectl commands, like `-f` means read from file not matter what command you use.
but if @kubernetes/sig-cli-maintainers all agree with this  `-m`, I have no objections.

Reply:
> Use short flags sparingly, only for the most frequently used options, prefer lowercase over uppercase for the most common cases, try to stick to well known conventions for UNIX commands and/or Docker, where they exist, and update this list when adding new short flags

This doesn't warrant a short-flag, in my opinion

Reply:
@liggitt @shiywang IIUC, you mean that `--message` is ok but `-m` is not, right?